Cards/MadGraph5_aMCatNLO/Z/Zto2Nu-2Jets_1J_PTNuNu-600_amcatnloFXFX-pythia8/Zto2Nu-2Jets_1J_PTNuNu-600_amcatnloFXFX-pythia8.json

```diff
@@ -11,7 +11,7 @@
         "set run_card ptj 10"
     ],
     "model_params_vars": {},
-    "fragment": ["Generator/ExternalLHEProducer_MadGraph5_aMCatNLO.dat", "PartonShower/amcatnloFXFX-pythia8.dat", "Filter/LHE_V_Pt600_filter.dat"],
+    "fragment": ["Generator/ExternalLHEProducer_MadGraph5_aMCatNLO.dat", "PartonShower/amcatnloFXFX-pythia8.dat", "Filter/LHE_V_Pt600to-1_filter.dat"],
```

Comment:
did we say by convention "to-1" should be dropped

Reply:
ah my mistake with the scripts thanks for catching it